pyproject.toml

```diff
@@ -26,7 +26,7 @@ classifiers = [
 ]
 dependencies = [
     "Sphinx>=4.2.0",
-    "pydata-sphinx-theme>=0.14.0",
+    "pydata-sphinx-theme==0.14.4",
```

Comment:
Should we be less restrictive by asking `<=0.14.4`?

Reply:
I'd agree here... pinning down a specific PyData Sphinx Theme version seems too much for me

Reply:
if we are not giving the specified version, dependabot will not open a PR right? but its ok, we can keep an eye on their release

Reply:
Dependabot should open a PR on upper limits as well (if I remember properly)